cipher-symmetric/src/main/scala/tsec/cipher/symmetric/imports/TripleDES.scala

```diff
@@ -0,0 +1,5 @@
+package tsec.cipher.symmetric.imports
+
+sealed trait TripleDES
+
+object TripleDES extends BlockCipherEV[TripleDES]("DESede", 8, 24)
```

Comment:
I didn't know `DESede` is a synonym for `TripleDES`. Maybe we should add a note here ?

Reply:
Eh, I'd actually do it backwards and leave TripleDES as the real name.

`DESede` means `DES encryption-decryption-encryption` which is a way to apply DES for encrypting things.

Reply:
thank you